src/app/kyb/page.tsx

```
@@ -0,0 +1,367 @@
+  { id: 4, name: 'Financial Information', completed: false },
+  { id: 5, name: 'Compliance', completed: false },
+  { id: 6, name: 'Complete', completed: false },
+];
+
+export default function KYBPage() {
+  const router = useRouter();
+  const [currentStep, setCurrentStep] = useState(1);
+  const [isSubmitted, setIsSubmitted] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+  const [isSubmitted, setIsSubmitted] = useState(false);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'isSubmitted'. Consider to delete it or rename it.

'isSubmitted' is defined here:


(lint/suspicious/noRedeclare)